Makefile

```diff
@@ -47,6 +48,7 @@ $(APP_NAME)-%: $(TARGET)-%
 	@mkdir -p $(APP_COMPLETIONS_DIR)
 	@gzip -c $(MANPAGE) > $(APP_EXTRAS_DIR)/alacritty.1.gz
 	@gzip -c $(MANPAGE-MSG) > $(APP_EXTRAS_DIR)/alacritty-msg.1.gz
+	@cp -fp $(CONFIGFILE) $(APP_EXTRAS_DIR)/
```

Comment:
Can you move this down to the other lines that are just `cp -fp`, just for consistency?